src/usb/usb.c

```diff
@@ -25,13 +25,14 @@
 #include "usb_processing.h"
 
 #ifndef TESTING
+#include <driver_init.h>
 #include <hal_timer.h>
+#include <u2f.h>
 #include <u2f/u2f_packet.h>
 #include <usb/usb_packet.h>
-extern struct timer_descriptor TIMER_0;
 #endif
 
-#define TIMEOUT_TICK_PERIOD_MS 100
+#define TIMEOUT_TICK_PERIOD_MS 50
```

Comment:
I wonder how this can impact other non-U2F operations.

Reply:
Looks like the value might reused here but without using a common definition?

https://github.com/digitalbitbox/bitbox02-firmware/blob/master/src/usb/usb_processing.c#L44

So that might need to be updated too.

Reply:
And this comment: https://github.com/digitalbitbox/bitbox02-firmware/blob/master/src/usb/usb_processing.c#L119